packages/components/src/Loader/Loader.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { PropsWithChildren, RefObject } from "react";
+import classnames from "classnames";
+import { Breakpoints, useResizeObserver } from "@jobber/hooks/";
+import styles from "./Loader.css";
+import { LoaderIndicator } from "./LoaderIndicator";
+
+type LoaderProps = IndeterminateLoaderProps | DeterminateLoaderProps;
+
+interface BaseLoaderProps {
+  /**
+   * Removes the overlay and has the component render in the flow of the layout
+   * @default false
+   */
+  readonly inline?: boolean;
+
+  /**
+   * Shows a spinner when false, progress bar when true.
+   * @default false
+   */
+  determinate?: boolean;
```

Comment:
The props table is showing these differently than I'd expect:

![image](https://user-images.githubusercontent.com/39704901/128229527-f3984086-e527-462a-870c-55e4e727b9d5.png)

Is it related to the interfaces being defined below?

Reply:
It looks like the Props of component doesn't deal well with union types. @eddysims what do you think is the best way to communicate the props, as I feel as though the union type works well in code editors but not well with this static prop table